client/src/partials/templates/bhLocationSelect.tmpl.html

```diff
@@ -3,11 +3,11 @@
   a child form is added for validation purposes, and to make the HTML easier
   to read.
 -->
-<fieldset ng-disabled="$ctrl.disable" ng-form="LocationSelectForm" data-bh-location-select>
+<fieldset ng-disabled="$ctrl.disable"  ng-form="{{$ctrl.name}}" ng-model="somevaluetwo" data-bh-location-select>
 
   <!-- allows a user to select a country -->
   <div class="form-group"
-    ng-class="{ 'has-error' : $ctrl.validationTrigger && LocationSelectForm.country.$invalid }"
+    ng-class="{ 'has-error' : $ctrl.validationTrigger && this[$ctrl.name].country.$invalid }"
```

Comment:
I wonder if using [ngInit](https://docs.angularjs.org/api/ng/directive/ngInit) would help clean the template up.  Someone who doesn't have all the context here wouldn't know what `this` was referring to.  Could you do something like this:

```html
<fieldset ng-disabled="$ctrl.disabled" ng-form="{{$ctrl.name}}" ng-init="LocationForm = this[$ctrl.name]" data-bh-location-select>

  <div class="form-group" ng-class="{ 'has-error' : $ctrl.validationTrigger && LocationForm.country.$invalid }">
  
  <!-- etc ... -->

</fieldset>
```
This way, you avoid having to know what `this` is, except at the top of the form.   If this pattern is possible, I think it is preferable.